pontoon/api/views.py

```diff
@@ -128,8 +126,6 @@ def get_user_actions(request, date, slug):
 
 
 class UserActionsView(APIView):
-    permission_classes = [IsAuthenticated]
```

Comment:
This requirement should remain - it's needed for performance reasons.

Reply:
This comment was marked as resolved, but the change is still in. Could you elaborate on this?

Reply:
Sorry, it seems like you just resolved the comment without addressing it for the 2nd time. 🤣 
Am I missing something?